scripts/dev-setup.sh

```
@@ -0,0 +1,233 @@
+    # govulncheck
+    print_status "Installing govulncheck..."
+    go install golang.org/x/vuln/cmd/govulncheck@latest
+
+    print_success "Development tools installed"
+}
+
+# Setup environment file
+setup_env() {
+    print_status "Setting up environment configuration..."
+
+    if [ \! -f .env ]; then
```

Shellcheck (0.10.0)

[warning] This expression is constant. Did you forget the $ on a variable?

(SC2050)